test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/LanguageServer/LanguageServiceTests.cs

```diff
@@ -404,6 +404,24 @@ public async Task CheckForNonTSqlLanguageTest(string scriptText, bool expectedRe
             var result = await langService.CheckForNonTSqlLanguage(scriptFile.ClientUri, parseResult);
 
             Assert.AreEqual(expectedResult, result, message);
+            return result;
+        }
+
+        [Test]
+        [TestCase(
+            "./TestScript.txt", 
+            false, 
+            "Long Sql Server script; should not detect Non-T-SQL syntax"
```

Comment:
May as well state that this "long SQL Server script" is "AdventureWorks DacFx deployment script" and name the .txt file appropriately.